features/mbedtls/targets/TARGET_STM/aes_alt.c

```diff
@@ -1,5 +1,5 @@
 /*
- *  Hardware aes collector for the STM32F4 family
+ *  Hardware aes collector for the STM32F4 F7 and L4 family
```

Comment:
I am terribly sorry, but I am not very familiar with this terminology, could you please give me some pointers on what an "AES collector" is?

Reply:
I honestly don't remember where I copied this expression from. I'll rephrase to `Hardware AES implementation for STM32F4 / STM32L4 and STM32F7 families`